README.md

```
@@ -365,3 +365,20 @@ Tests are tagged with the following labels to determine their execution suite:
 - **`[PreferredHost][Serial]`** - Tests that verify KubeScheduler communication with kube-apiserver over preferred host
 
 For more information about the OTE framework, see the [openshift-tests-extension documentation](https://github.com/openshift-eng/openshift-tests-extension).
+
+
+
+
+
+  # Set KUBECONFIG
```

markdownlint-cli2 (0.21.0)

[warning] Headings must start at the beginning of the line

(MD023, heading-start-left)